egs/gale_arabic/s5b/run.sh

```diff
@@ -0,0 +1,167 @@
+
+# LM training
+local/gale_train_lms.sh || exit 1;
+
+local/gale_format_data.sh  || exit 1;
+# G compilation, check LG composition
+
+# Now make MFCC features.
+# mfccdir should be some place with a largish disk where you
+# want to store MFCC features.
+mfccdir=mfcc
+
+for x in train test ; do
+  steps/make_mfcc.sh --cmd "$train_cmd" --nj $nJobs \
+    data/$x exp/make_mfcc/$x $mfccdir
+  utils/fix_data_dir.sh data/$x # some files fail to get mfcc for many reasons
+  steps/compute_cmvn_stats.sh data/$x exp/make_mfcc/$x $mfccdir
+done
+
+
+# Here we start the AM
+
+# Let's create a subset with 10k segments to make quick flat-start training:
+utils/subset_data_dir.sh data/train 10000 data/train.10K || exit 1;
+
+# Train monophone models on a subset of the data, 10K segment
+# Note: the --boost-silence option should probably be omitted by default
+steps/train_mono.sh --nj 40 --cmd "$train_cmd" \
+  data/train.10K data/lang exp/mono || exit 1;
+
+
+# Get alignments from monophone system.
+steps/align_si.sh --nj $nJobs --cmd "$train_cmd" \
+  data/train data/lang exp/mono exp/mono_ali || exit 1;
+
+# train tri1 [first triphone pass]
+steps/train_deltas.sh --cmd "$train_cmd" \
+  2500 30000 data/train data/lang exp/mono_ali exp/tri1 || exit 1;
+
+# First triphone decoding
+utils/mkgraph.sh data/lang_test exp/tri1 exp/tri1/graph
+steps/decode.sh  --nj $nDecodeJobs --cmd "$decode_cmd" \
+  exp/tri1/graph data/test exp/tri1/decode
+  
+steps/align_si.sh --nj $nJobs --cmd "$train_cmd" \
+  data/train data/lang exp/tri1 exp/tri1_ali || exit 1;
+
+# Train tri2a, which is deltas+delta+deltas
+steps/train_deltas.sh --cmd "$train_cmd" \
+  3000 40000 data/train data/lang exp/tri1_ali exp/tri2a || exit 1;
+
+# tri2a decoding
+utils/mkgraph.sh data/lang_test exp/tri2a exp/tri2a/graph
+steps/decode.sh --nj $nDecodeJobs --cmd "$decode_cmd" \
+  exp/tri2a/graph data/test exp/tri2a/decode
+
+# train and decode tri2b [LDA+MLLT]
+steps/train_lda_mllt.sh --cmd "$train_cmd" 4000 50000 \
+  data/train data/lang exp/tri1_ali exp/tri2b || exit 1;
+
+utils/mkgraph.sh data/lang_test exp/tri2b exp/tri2b/graph
+steps/decode.sh --nj $nDecodeJobs --cmd "$decode_cmd" \
+  exp/tri2b/graph data/test exp/tri2b/decode
+
+# Align all data with LDA+MLLT system (tri2b)
+steps/align_si.sh --nj $nJobs --cmd "$train_cmd" \
+  --use-graphs true data/train data/lang exp/tri2b exp/tri2b_ali  || exit 1;
+
+
+# From 2b system, train 3b which is LDA + MLLT + SAT.
+steps/train_sat.sh --cmd "$train_cmd" \
+  5000 100000 data/train data/lang exp/tri2b_ali exp/tri3b || exit 1;
+
+utils/mkgraph.sh data/lang_test exp/tri3b exp/tri3b/graph
+steps/decode_fmllr.sh --nj $nDecodeJobs --cmd \
+  "$decode_cmd" exp/tri3b/graph data/test exp/tri3b/decode
+
+# From 3b system, align all data.
+steps/align_fmllr.sh --nj $nJobs --cmd "$train_cmd" \
+  data/train data/lang exp/tri3b exp/tri3b_ali || exit 1;
+  
+
+# nnet3 cross-entropy 
+local/nnet3/run_tdnn.sh #tdnn recipe:
```

Comment:
I'd recommend to have most of these commented out in the run.sh by default, and just run the best of these recipes.